src/compiler/scala/tools/nsc/ast/TreeDSL.scala

```diff
@@ -79,13 +78,9 @@ trait TreeDSL {
       def INT_-   (other: Tree)     = fn(target, getMember(IntClass, nme.MINUS), other)
 
       // generic operations on ByteClass, IntClass, LongClass
-      @unused("avoid warning for multiple parameters")
       def GEN_|   (other: Tree, kind: ClassSymbol)  = fn(target, getMember(kind, nme.OR), other)
-      @unused("avoid warning for multiple parameters")
       def GEN_&   (other: Tree, kind: ClassSymbol)  = fn(target, getMember(kind, nme.AND), other)
-      @unused("avoid warning for multiple parameters")
       def GEN_==  (other: Tree, kind: ClassSymbol)  = fn(target, getMember(kind, nme.EQ), other)
-      @unused("avoid warning for multiple parameters")
```

Comment:
I think this was due to an early version of multiarg infix that warned for this name, but it doesn't seem to warn now.